src/lib/mocked-apps.js

```diff
@@ -90,48 +97,70 @@ export default [
       pt_br: 'Outros',
       en_us: 'Others'
     }),
-    apps: [{
+    apps: [
+     {
+      title: 'Sak',
+      icon: iconSak,
+      short_description: 'O Sak é um aplicativo de automação de envios de mensagem exclusivo, a recuperação de carrinhos abandonados e pedidos não pagos.',
+      clicked () {
+        openNewTab('https://get.sak.com.br/')
+      }
+    },
+    {
+      title: 'Rastreio.net',
+      icon: iconRastreionet,
+      short_description: 'O Rastreio.net é um aplicativo que facilita o processo de espera do cliente que aguarda seu pedido possibilitando o rastreio.',
+      clicked () {
+        openNewTab('https://get.rastreio.net/')
+      }
```

Comment:
Uai mas simplesmente linka pro site do caras?
Fica meio inútil o CTA do app aí, não? Tem que ter um contato mais direto ou pelo menos uma página que mencione a E-Com Plus e indique o que o lojista precisa fazer pra integrar...

Reply:
> Uai mas simplesmente linka pro site do caras?

Então, perguntei pra onde linkar, falaram para deixar a home mesmo.

Reply:
Uai mas aí não tô falando do desejo do cara não 😄 
Não sei se faz sentido colocar no admin simplesmente um link pra home de um produto alheio que inclusive integra (e fala sobre) outras plataformas...

Reply:
Lembrando que isso aí aparece só no painel inclusive, quer dizer, isso nem vai nos ajudar a converter alguém que queira a integração, só tá adicionando no nosso painel um link pra página de captura dos outros...
Sem nenhuma integração automática, contato direto útil, link de indicação (com share pra cadastro ou algo do tipo) ou pelo menos um artigo pra nossa plataforma, é só propaganda grátis pra esse pessoal aí 😄

Reply:
Mas pelo que eu entendi tem a integração do lado deles. Ao menos colocam nossa logo na home por lá. Tavam reclamando de terem feito a integração e poucos sellers usam. Eles tem outros apps que se pá integrem se conseguirem converter outros clientes por aqui no que já tem integrado. 

Claro que com um revenue share é melhor, e tendo uma lp prórpira nossa por lá também. Por enquanto fizeram só esse [vídeo](https://www.youtube.com/watch?v=JZkxTj8iOmA) mostrando como instalar. Eles já tiveram o custo da integração também né... Eu acho que só temos a ganhar colocando no painel, além de mostrar que a plataforma tem mais ferramentas, ajuda a estreitar relações com essa galera, mas se vc achar que não vale a pena me fala que eu aviso pro cara aqui que não vai rolar.

Reply:
> além de mostrar que a plataforma tem mais ferramentas

> Lembrando que isso aí aparece só no painel inclusive, quer dizer, isso nem vai nos ajudar a converter alguém que queira a integração

Se sua intenção é mostrar que tem mais ferramentas isso deveria estar no site https://www.e-com.plus/todas-integracoes-que-sua-loja-virtual-precisa-e-com-plus , esses mocks no painel aí a intenção realmente é ajudar a configurar né

---

> Por enquanto fizeram só esse [vídeo](https://www.youtube.com/watch?v=JZkxTj8iOmA) mostrando como instalar.

Esse vídeo aí é BEM melhor que a LP deles pra estar no painel, não?

Reply:
Pode ser. Mas acho o link pra eles importante também, pra facilitar quem quiser conhecer a parada. 
Talvez coloca o link pra eles na descrição e o vídeo no botão instalar.

Reply:
Mas nesse caso é só um link "entre em contato" acho que faz mais sentido linkar o site como pediram do que o tutorial no youtube,

Reply:
Aí era só mudar o texto do botão...
Mas enfim, entendi que seu principal objetivo nesse caso é realmente fazer propaganda pra eles 🙃 
De novo, acho que deveria colocar esses apps no site, porque aí pelo menos faz pra nós um pouco também.
No painel a função costuma ser ajudar a configurar a ferramenta que ele já quer usar, não estimular o lojista a contratar uma ferramenta nova.

Reply:
Coloquei lá no site também. Tanto essas duas novas quanto as outras que estavam no painel sem icone.